### ISSUE TITLE
Can't read sensors on a Thinkpad

### ISSUE THREAD

**Issue by ntoshev** [nicolargo/glances#711]
On my laptop, glances doesn't show sensors information. In glances.log I have:
```
2015-11-03 11:30:50,164 -- INFO -- Start Glances 2.5.1
2015-11-03 11:30:50,164 -- INFO -- CPython 2.7.6 and PSutil 3.2.2 detected
2015-11-03 11:30:50,168 -- INFO -- Start standalone mode
2015-11-03 11:30:50,176 -- WARNING -- PyStache lib not installed (action script with mustache will not work)
2015-11-03 11:30:50,233 -- ERROR -- Cannot grab sensors temperatures (Can't read)
2015-11-03 11:30:50,236 -- ERROR -- Cannot grab FAN speed (Can't read)
2015-11-03 11:30:50,433 -- ERROR -- Cannot grab sensors temperatures (Can't read)
2015-11-03 11:30:50,436 -- ERROR -- Cannot grab FAN speed (Can't read)
2015-11-03 11:30:53,608 -- ERROR -- Cannot grab sensors temperatures (Can't read)
```
On the command line it works:
```
$ sensors
acpitz-virtual-0
Adapter: Virtual device
temp1:        +43.0°C  (crit = +127.0°C)
temp2:        +45.0°C  (crit = +100.0°C)

thinkpad-isa-0000
Adapter: ISA adapter
fan1:        1963 RPM
temp1:        +43.0°C  
temp2:        +45.0°C  
temp3:        +38.0°C  
temp4:            N/A  
temp5:        +27.0°C  
temp6:            N/A  
temp7:        +27.0°C  
temp8:            N/A  
temp9:        +41.0°C  
temp10:       +48.0°C  
temp11:       +46.0°C  
temp12:           N/A  
temp13:           N/A  
temp14:           N/A  
temp15:           N/A  
temp16:           N/A  

coretemp-isa-0000
Adapter: ISA adapter
Core 0:       +45.0°C  (high = +105.0°C, crit = +105.0°C)
Core 1:       +43.0°C  (high = +105.0°C, crit = +105.0°C)
```

When I try Py3Sensors directly, I get:
```python
for chip in sensors.iter_detected_chips():
...         print ('%s at %s' % (chip, chip.adapter_name))
...         for feature in chip:
...             print ('  %s: %.2f' % (feature.label, feature.get_value()))
acpitz-virtual-0 at Virtual device
  temp1: 44.00
  temp2: 45.00
thinkpad-isa-0000 at ISA adapter
  fan1: 1965.00
  temp1: 44.00
  temp2: 44.00
  temp3: 38.00
Traceback (most recent call last):
  File "<pyshell#23>", line 4, in <module>
    print ('  %s: %.2f' % (feature.label, feature.get_value()))
  File "/usr/local/lib/python2.7/dist-packages/sensors/api4.py", line 85, in get_value
    return next(iter(self)).get_value()
  File "/usr/local/lib/python2.7/dist-packages/sensors/api4.py", line 36, in get_value
    _get_value(byref(self.parent.chip), self.number, byref(result))
  File "/usr/local/lib/python2.7/dist-packages/sensors/common.py", line 27, in _error_check
    raise SensorsError(_strerror(result), result)
SensorsError: Can't read
```
But when I ignore exceptions, the useful data is there:
```python
for chip in sensors.iter_detected_chips():
...         print ('%s at %s' % (chip, chip.adapter_name))
...         for feature in chip:
...             try:
...                 print ('  %s: %.2f' % (feature.label, feature.get_value()))
...             except Exception as e :
...                 print e.message
acpitz-virtual-0 at Virtual device
  temp1: 46.00
  temp2: 45.00
thinkpad-isa-0000 at ISA adapter
  fan1: 1979.00
  temp1: 46.00
  temp2: 45.00
  temp3: 38.00
Can't read
  temp5: 27.00
Can't read
  temp7: 27.00
Can't read
  temp9: 41.00
  temp10: 48.00
  temp11: 46.00
Can't read
Can't read
Can't read
Can't read
Can't read
coretemp-isa-0000 at ISA adapter
  Core 0: 46.00
  Core 1: 43.00
```

**Comment by nicolargo** [comment]
Should be corrected in the develop branch. Can you test it ?

**Comment by ntoshev** [comment]
I get a stack trace:

```~/glances$ ((detached from 9417407)) glances
Traceback (most recent call last):
  File "/usr/local/bin/glances", line 9, in <module>
    load_entry_point('Glances==2.5.1', 'console_scripts', 'glances')()
  File "/usr/local/lib/python2.7/dist-packages/Glances-2.5.1-py2.7.egg/glances/__init__.py", line 122, in main
    args=core.get_args())
  File "/usr/local/lib/python2.7/dist-packages/Glances-2.5.1-py2.7.egg/glances/core/glances_standalone.py", line 68, in __init__
    self.stats.update()
  File "/usr/local/lib/python2.7/dist-packages/Glances-2.5.1-py2.7.egg/glances/core/glances_stats.py", line 165, in update
    self._plugins[p].update()
  File "/usr/local/lib/python2.7/dist-packages/Glances-2.5.1-py2.7.egg/glances/plugins/glances_hddtemp.py", line 64, in update
    self.stats = self.glancesgrabhddtemp.get()
  File "/usr/local/lib/python2.7/dist-packages/Glances-2.5.1-py2.7.egg/glances/plugins/glances_hddtemp.py", line 146, in get
    self.__update__()
  File "/usr/local/lib/python2.7/dist-packages/Glances-2.5.1-py2.7.egg/glances/plugins/glances_hddtemp.py", line 121, in __update__
    temperature = float(fields[offset + 3])
ValueError: could not convert string to float: ERR
```

**Comment by asergi** [comment]
Fixed b7f2ff8.
